### ISSUE TITLE
Integrate the Commonwealth of Britain Bill proposals

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#119]
There is a lot of good stuff in the [Commonwealth of Britain Bill](https://en.wikipedia.org/wiki/Commonwealth_of_Britain_Bill) proposed in 1991 by Tony Benn that would fit well here. We have some already, but there's probably more we could bring in.

See the [ampp3d coverage](http://ampp3d.mirror.co.uk/2014/03/14/how-close-are-we-to-tony-benns-vision-of-democracy/) for a simple overview.

**Comment by andrewdwilliams** [comment]
I like the thought behind the bill, but I can't actually find the text of the bill anywhere online. If I do, I'll drop a link here.

**Comment by Floppy** [comment]
Yeah, I never found the actual thing. I imagine we'd treat it like a shopping list of proposals to write up and include.
